packages/amplication-code-gen-types/src/plugin-events-params.ts

```diff
@@ -55,6 +55,25 @@ export interface CreateEntityControllerBaseParams extends EventParams {
   serviceId: namedTypes.Identifier;
 }
 
+export interface CreateEntityControllerGrpcParams extends EventParams {
+  template: namedTypes.File;
+  entityName: string;
+  entityServiceModule: string;
+  templateMapping: { [key: string]: any };
+  controllerBaseId: namedTypes.Identifier;
+  serviceId: namedTypes.Identifier;
+}
+export interface CreateEntityControllerGrpcBaseParams extends EventParams {
```

Comment:
These events seem really binding to only 1 plugin that is the opposite of what a plugin event should be.

What was the reasoning behind this instead of adding the new files from the plugin?

Reply:
This was the agreed design in this case, and it's not binding to a single plugin since any plugin can have a systemSetting of "generateGRPC=true" and use these events.